apps/portals/src/configurations/elportal/synapseConfigs/handleFilesToParticipants.tsx

```diff
@@ -6,14 +6,14 @@ import {
   SynapseClient,
   SynapseConstants,
   SynapseUtilityFunctions,
+  CustomControlCallbackData,
 } from 'synapse-react-client'
-import { CustomControlCallbackData } from 'synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControls'
```

Comment:
This is a type import so it was removed from compilation, so it happened to work in this case, but we should not import from direct paths